src/hrflow_connectors/connectors/workable/actions.py

```diff
@@ -2,14 +2,15 @@
 from pydantic import Field
 import html
```

Comment:
Same as the comment for greenhouse connector: Remove unnecessary import html because we already have it now in `remove_html_tags` function

Reply:
`html.unescape` is not used in this file and it is not in `remove_html_tags`. In any case, you are right. So, i remove that import.